src/pcost.py

```diff
@@ -11,12 +12,16 @@ def portfolio_cost(file_path: Union[str, Path]) -> float:
     :param file_path: Path to the portfolio file
     :return: The cost of the portfolio
     """
-    portfolio = parse_csv(
+    portdict = parse_csv(
         file_path,
         select=["shares", "price"],
         convert_fn={"shares": int, "price": float},
     )
-    portfolio = cast(List[Dict[str, Any]], portfolio)
-    total_cost = sum([p["shares"] * p["price"] for p in portfolio])
+    portfolio = [
+        Stock(name="", shares=pd["shares"], price=pd["price"])
+        for pd in portdict
+        if isinstance(pd, dict)
```

Comment:
Why do you need to filter here?

Reply:
I was getting a mypy error for line 21 and adding the filter seemed to fix it.
```
src/pcost.py:21: error: No overload variant of "__getitem__" of "tuple" matches argument type "str"
```

Reply:
Adding `portdict: List[Dict[str, Any]] = parse_csv(...)` also causes an error but I can fix it with:
```python
portdict = parse_csv(...)
portdict = cast(List[Dict[str, Any]], portdict)
```
Is this ok?

Reply:
No it's OK to keep this filtering!